contracts/community-tokens/contracts/BaseToken.sol

```diff
@@ -3,14 +3,14 @@ pragma solidity ^0.8.17;
 
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
 import "@openzeppelin/contracts/utils/Context.sol";
 import "@openzeppelin/contracts/utils/Counters.sol";
 
-contract CollectibleV1 is
+abstract contract BaseToken is
     Context,
-    ERC721Enumerable,
-    Ownable
+    ERC721Enumerable
 {
     using Counters for Counters.Counter;
```

Comment:
`Counter` is no longer part of the latest OpenZeppelin as of its latest release.
Here's the issue that removed it: https://github.com/OpenZeppelin/openzeppelin-contracts/issues/4233

For future versions of this contract, we might want to consider dropping the library altogether